Docs/Configuration.tex

```diff
@@ -3632,34 +3661,34 @@ \subsection{Security Properties}\label{miscsecurityprops}
 
   \begin{itemize}
   \tightlist
-  \item \texttt{Disabled} --- loading DMG images will fail. \texttt{Disabled}
-    policy will still let macOS Recovery to load in most cases as there
-    usually are \texttt{boot.efi} files compatible with Apple Secure Boot.
+  \item \texttt{Disabled} --- loading DMG images will fail. The \texttt{Disabled}
+    policy will still let the macOS Recovery load in most cases as typically,
+    there are \texttt{boot.efi} files compatible with Apple Secure Boot.
     Manually downloaded DMG images stored in \texttt{com.apple.recovery.boot}
     directories will not load, however.
-  \item \texttt{Signed} --- only Apple-signed DMG images will load. Due to
-    Apple Secure Boot design \texttt{Signed} policy will let any Apple-signed
-    macOS Recovery to load regardless of Apple Secure Boot state, which may
+  \item \texttt{Signed} --- only Apple-signed DMG images will load. Due to the
```

Comment:
I saw dmg instead of DMG in several places, could you fix?

Reply:
Seems there was only one or two relevant ones